packaging/installer/README.md

```diff
@@ -545,21 +545,21 @@ Move these files to the `/tmp` directory on the offline system in whichever way
 your defined policy allows (if any).
 
 Now you can run either the `kickstart.sh` or `kickstart-static64.sh` scripts
-using the `--local-tarball-override` option. This option requires you to specify
+using the `--local-files` option. This option requires you to specify
 the location and names of the files you just downloaded. 
 
-!!! note When using `--local-tarball-override`, the `kickstart.sh` or
+!!! note When using `--local-files`, the `kickstart.sh` or
     `kickstart-static64.sh` scripts won't download any Netdata assets from the
     internet. But, you may still need a connection to install dependencies using
     your system's package manager. The scripts will warn you if your system
     doesn't have all the dependencies.
 
 ```bash
 # kickstart.sh
-bash &lt;(curl -Ss https://my-netdata.io/kickstart.sh) --local-tarball-override /tmp/netdata.tar.gz /tmp/checksums.txt /tmp/go.d.binary.tar.gz /tmp/go.d.config.tar.gz
+bash &lt;(curl -Ss https://my-netdata.io/kickstart.sh) --local-files /tmp/netdata.tar.gz /tmp/checksums.txt /tmp/go.d.binary.tar.gz /tmp/go.d.config.tar.gz
```

Comment:
Forgive my ignorance; but presumably this won't work if we're working on a disconnected environment? Shouldn't we be referencing the bash file in packaging/installer/ rather than the one online?

Reply:
well, we give the users the shortcut to directly execute the above command, but in the offline mode they will just download and execute it manually.

It may make sense to add a note though

Reply:
Actually, you just pinpointed something without realising it.
Kickstart has the dependency script that is running directly from the web, that will also need to be present for the process to work offline

Reply:
coming back on the instructions, I don't know if we should be adjusting them to make them completely guided. To me it kind of makes sense that the user can simply download and run the kickstart if he can't directly execute the proposed command, otherwise we make our instructions too verbose imho

I think i would need commends from @joelhans on this one

Reply:
Sure, as you deem fit. I think I’m just coming from a lazy perspective (although technical enough to figure out I can’t run those instructions). I would hope to be able to just copy and paste the instructions in the offline install section. In order for me to do this offline install I would love to:
1. Checkout / download master branch
2. Execute one bash file that will download all of my temporary files I need (possibly into an ‘offline install’ folder rather than /tmp) as well as copying any kickstart bash scripts to that folder. 
3. Copy that folder to the non internet machine.
4. Execute one bash script to install netdata using the pre-downloaded files

Might be too much for now, and I can def try and do this myself and contribute, but that would be my best case scenario. :)

Reply:
That is a good point actually, it just increases the scope of the change.
You are basically asking for a preparation script, which is a layer on top of the offline install of kickstart.

This script will gather all required elements for an offline install basically.
The `offline-install.sh` script would execute the following steps with the `--prepare` switch:

1) Download all required netdata items that kickstart now needs in a folder, including the kickstart.sh script
2) Download all needed binary packages into a folder from the respective repo (yum, apt, etc)

Then using the `--install` switch, it would:
1) Install offline packages to the system
2) run the kickstart with the right flags.


I think i should create a new issue for this, so that we can prioritise and do it.
It needs some thinking to cut the right corners